webrtc-stats.html

```diff
@@ -1860,6 +1881,168 @@ <h2>
+          <code>RTCVideoSourceStats</code>. The <code>type</code> is the same
+          (<code>"media-source"</code>) but <code>kind</code> is different (<code>"audio"</code>
+          or <code>"video"</code>) depending on the kind of track.
+        </p>
+        <p>
+          The media source stats objects are created when a track is attached to any
+          <code>RTCRtpSender</code> and may subsequently be attached to multiple senders during
+          its life. The life of this object ends when the track is no longer attached to any
+          sender of the same <code>RTCPeerConnection</code>. If a track whose media source
+          object ended is attached again this results in a new media source stats object whose
+          counters (such as number of frames) are reset.
+        </p>
+        <div>
+          <pre class="idl">dictionary RTCMediaSourceStats : RTCStats {
+             DOMString       trackIdentifier;
+             DOMString       kind;
+};</pre>
+          <section>
+            <h2>
+              Dictionary <a class="idlType">RTCMediaSourceStats</a> Members
+            </h2>
+            <dl data-link-for="RTCMediaSourceStats" data-dfn-for="RTCMediaSourceStats"
+            class="dictionary-members">
+              <dt>
+                <dfn><code>trackIdentifier</code></dfn> of type <span class=
+                "idlMemberType">DOMString</span>
+              </dt>
+              <dd>
+                <p>
+                  The value of the <code>MediaStreamTrack</code>'s
+                  <code>id</code> attribute.
+                </p>
+              </dd>
+              <dt>
+                <dfn><code>kind</code></dfn> of type <span class=
+                "idlMemberType">DOMString</span>
+              </dt>
+              <dd>
+                <p>
+                  The value of the <code>MediaStreamTrack</code>'s
+                  <code>kind</code> attribute. This is either
+                  <code>"audio"</code> or <code>"video"</code>. If it is
+                  <code>"audio"</code> then this stats object is of type
+                  <code>RTCAudioSourceStats</code>. If it is
+                  <code>"video"</code> then this stats object is of type
+                  <code>RTCVideoSourceStats</code>.
+                </p>
+              </dd>
+            </dl>
+          </section>
+        </div>
+      </section>
+      <section id="audiosourcestats-dict*">
+        <h3>
+          RTCAudioSourceStats dictionary
+        </h3>
+        <p>
+          The <dfn>RTCAudioSourceStats</dfn> dictionary represents an audio
+          track that is attached to one or more senders. It is an
+          <code><a>RTCMediaStreamSource</a></code> whose <code>kind</code> is
+          <code>"audio"</code>.
+        </p>
+        <div>
+          <pre class="idl">dictionary RTCAudioSourceStats : RTCMediaSourceStats {
+};</pre>
```

Comment:
confirming we currently have no stats for audio source.

Reply:
We may want to put stuff like input energy levels here, but no, currently there's nothing here.